frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,42 @@
-import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
+import React, {useState, useEffect} from 'react'
```

Comment:
don't need to import React anymore

Reply:
If I remove React, the app doesn't compile. I believe it need React to render the JSX

Reply:
ahh youre right, nvm then :stuck_out_tongue: